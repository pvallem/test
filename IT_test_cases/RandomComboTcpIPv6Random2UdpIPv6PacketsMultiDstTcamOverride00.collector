bidir_flow_info {
  key {
    src_address: "\001\000\000\000\000\000\000\000\000\000\000\000\000\000\000e"
    dst_address: "\001\002\003\004\000\000\000\000\000\000\000\000\000\000\000\000"
    src_port: 222
    dst_port: 222
    proto: 17
    tenant_id: 10
    flow_start_time: 1388563200000030
    key_type: IPV6
  }
  reverse_flow_info {
    start_time: 1388563200000030
    ip_info {
      qos: 0
    }
    flow_features {
      packet_count: 1
    }
    end_time: 1388563200000030
    sensor_id: "127.0.0.1:5675"
  }
}
bidir_flow_info {
  key {
    src_address: "\001\000\000\000\000\000\000\000\000\000\000\000\000\000\000d"
    dst_address: "\001\002\003\004\000\000\000\000\000\000\000\000\000\000\000\000"
    src_port: 222
    dst_port: 222
    proto: 17
    tenant_id: 10
    flow_start_time: 1388563200000010
    key_type: IPV6
  }
  reverse_flow_info {
    start_time: 1388563200000010
    ip_info {
      qos: 0
    }
    flow_features {
      packet_count: 1
    }
    end_time: 1388563200000010
    sensor_id: "127.0.0.1:5675"
  }
}
bidir_flow_info {
  key {
    src_address: "(\227\236N\265\221\216\302+\226\2160\034<zI"
    dst_address: "2m\r\302B\305V\\\'l\277\216\376\023\373d"
    src_port: 22252
    dst_port: 25439
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200000020
    key_type: IPV6
  }
  flow_info {
    start_time: 1388563200000020
    ip_info {
      last_cached_ttl: 8
      qos: 0
    }
    tcp_info {
      sequence_num: 10
      null_flags: 1
      syn_fin_rst_ack_all_zero: 1
      option_len_bins {
      }
    }
    flow_features {
      packet_count: 1
    }
    end_time: 1388563200000020
    sensor_id: "127.0.0.1:5675"
  }
}
bidir_flow_info {
  key {
    src_address: "=\251\366 u\246\372\254\332\222E\237\312\346\326\016"
    dst_address: "\262-\353\236\220P\227F+\307\232(\223\343\200l"
    src_port: 48262
    dst_port: 16135
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200000040
    key_type: IPV6
  }
  reverse_flow_info {
    start_time: 1388563200000040
    ip_info {
      last_cached_ttl: 8
      qos: 0
    }
    tcp_info {
      sequence_num: 20
      null_flags: 1
      syn_fin_rst_ack_all_zero: 1
      option_len_bins {
      }
    }
    flow_features {
      packet_count: 1
    }
    end_time: 1388563200000040
    sensor_id: "127.0.0.1:5675"
  }
}
bidir_flow_info {
  key {
    src_address: "r\217\254;\234\033\341\347\315\274D\265\257\036\222\224"
    dst_address: "u\263\367\312\246|\256z\031\261\362H\264{\005~"
    src_port: 19301
    dst_port: 55609
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200000000
    key_type: IPV6
  }
  reverse_flow_info {
    start_time: 1388563200000000
    ip_info {
      last_cached_ttl: 8
      qos: 0
    }
    tcp_info {
      sequence_num: 0
      null_flags: 1
      syn_fin_rst_ack_all_zero: 1
      option_len_bins {
      }
    }
    flow_features {
      packet_count: 1
    }
    end_time: 1388563200000000
    sensor_id: "127.0.0.1:5675"
  }
}
