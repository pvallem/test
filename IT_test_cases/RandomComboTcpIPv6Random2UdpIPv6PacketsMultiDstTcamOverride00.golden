flow_info {
  key {
    src_address: "u\263\367\312\246|\256z\031\261\362H\264{\005~"
    dst_address: "r\217\254;\234\033\341\347\315\274D\265\257\036\222\224"
    src_port: 55609
    dst_port: 19301
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200000000
    key_type: IPV6
  }
  start_time: 1388563200000000
  ip_info {
    last_cached_ttl: 8
    qos: 0
  }
  tcp_info {
    sequence_num: 0
    null_flags: 1
    syn_fin_rst_ack_all_zero: 1
    option_len_bins {
    }
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200000000
}
flow_info {
  key {
    src_address: "\001\002\003\004\000\000\000\000\000\000\000\000\000\000\000\000"
    dst_address: "\001\000\000\000\000\000\000\000\000\000\000\000\000\000\000d"
    src_port: 222
    dst_port: 222
    proto: 17
    tenant_id: 10
    flow_start_time: 1388563200000010
    key_type: IPV6
  }
  start_time: 1388563200000010
  ip_info {
    qos: 0
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200000010
}
flow_info {
  key {
    src_address: "(\227\236N\265\221\216\302+\226\2160\034<zI"
    dst_address: "2m\r\302B\305V\\\'l\277\216\376\023\373d"
    src_port: 22252
    dst_port: 25439
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200000020
    key_type: IPV6
  }
  start_time: 1388563200000020
  ip_info {
    last_cached_ttl: 8
    qos: 0
  }
  tcp_info {
    sequence_num: 10
    null_flags: 1
    syn_fin_rst_ack_all_zero: 1
    option_len_bins {
    }
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200000020
}
flow_info {
  key {
    src_address: "\001\002\003\004\000\000\000\000\000\000\000\000\000\000\000\000"
    dst_address: "\001\000\000\000\000\000\000\000\000\000\000\000\000\000\000e"
    src_port: 222
    dst_port: 222
    proto: 17
    tenant_id: 10
    flow_start_time: 1388563200000030
    key_type: IPV6
  }
  start_time: 1388563200000030
  ip_info {
    qos: 0
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200000030
}
flow_info {
  key {
    src_address: "\262-\353\236\220P\227F+\307\232(\223\343\200l"
    dst_address: "=\251\366 u\246\372\254\332\222E\237\312\346\326\016"
    src_port: 16135
    dst_port: 48262
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200000040
    key_type: IPV6
  }
  start_time: 1388563200000040
  ip_info {
    last_cached_ttl: 8
    qos: 0
  }
  tcp_info {
    sequence_num: 20
    null_flags: 1
    syn_fin_rst_ack_all_zero: 1
    option_len_bins {
    }
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200000040
}
